caffe2/operators/segment_reduction_op.h

```diff
@@ -1615,7 +1615,7 @@ class AbstractLengthsGradientOp : public Operator<Context> {
   }
 
   // Input layout:
-  //   orig_arg1, orig_arg2, ..., orig_argN, SEGMENT_GRADS, SEGMENT_IDS
+  //   orig_arg1, orig_arg2, ..., orig_argN, SEGMENT_GRADS, LENGTHS, INDICES
```

Comment:
is INDICES optional/conditional? Should it be "[INDICES]" in the comment like it is further below in the file?

Reply:
INDICES is not optional (required for GPU async version) even though it's not actually used in CPU version.

Reply:
INDICES is not optional (required for GPU async version) even though it's not used in CPU version.